src/main/java/axiomq/com/starwars/services/implementations/UserServiceImpl.java

```diff
@@ -49,4 +48,7 @@ public void deleteUser(Long userId) {
         userRepository.delete(user);
 
     }
+
+
```

Comment:
try to remove unnecessary spaces. ctrl+alt+L command for refactoring code I think